### PR TITLE
New endpoint Get Transaction interactions

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [x] No, but I am going to.

New endpoint - GET Transaction interactions - Payments Gateway API (EDU-11393)

### CODE REVIEW

File: VTEX - Payments Gateway API.json [hunk 1]
```diff
@@ -4411,7 +4584,341 @@
...
+                            "properties": {
+                                "dueDateType": {
+                                    "type": "number",
+                                    "description": "Indicates whether the billing date will be at the end (0) or beginning of the period (1).",
+                                    "enum": [ 0, 1]
+                                },
+                                "interestRateMethod": {
+                                    "type": "number",
+                                    "description": "Indicates the type of interest to calculate the value of the installments (Compound Interest = `null` or `0`, Simple Interest Rate with Tax = `1` and Single Interest = `2`).",
+                                    "enum": [ null, 0, 1, 2],
```
Comment (karenkrieger) [inline VTEX - Payments Gateway API.json:4657]:
```suggestion
                                    "enum": [null, 0, 1, 2],
```

File: VTEX - Payments Gateway API.json [hunk 2]
```diff
@@ -4472,12 +4982,12 @@
                             "dueDateType": {
                                 "type": "number",
                                 "description": "Indicates whether the billing date will be at the end (0) or beginning of the period (1).",
-                                "enum": ["0", "1"]
+                                "enum": [ 0, 1]
```
Comment (karenkrieger) [inline VTEX - Payments Gateway API.json:4985]:
```suggestion
                                "enum": [0, 1]
```

File: VTEX - Payments Gateway API.json [hunk 3]
```diff
@@ -4472,12 +4982,12 @@
...
                             },
                             "interestRateMethod": {
                                 "type": "number",
                                 "description": "Indicates the type of interest to calculate the value of the installments (Compound Interest = `null` or `0`, Simple Interest Rate with Tax = `1` and Single Interest = `2`).",
-                                "enum": [ "null", "0", "1", "2"],
+                                "enum": [ null, 0, 1, 2],
```
Comment (karenkrieger) [inline VTEX - Payments Gateway API.json:4990]:
```suggestion
                                "enum": [null, 0, 1, 2],
```

File: VTEX - Payments Gateway API.json [hunk 4]
```diff
@@ -4768,12 +5308,12 @@
                             "dueDateType": {
                                 "type": "number",
                                 "description": "Indicates whether the billing date will be at the end (0) or beginning of the period (1).",
-                                "enum": ["0", "1"]
+                                "enum": [ 0, 1]
```
Comment (karenkrieger) [inline VTEX - Payments Gateway API.json:5311]:
```suggestion
                                "enum": [0, 1]
```

File: VTEX - Payments Gateway API.json [hunk 5]
```diff
@@ -4768,12 +5308,12 @@
...
                             },
                             "interestRateMethod": {
                                 "type": "number",
                                 "description": "Indicates the type of interest to calculate the value of the installments (Compound Interest = `null` or `0`, Simple Interest Rate with Tax = `1` and Single Interest = `2`).",
-                                "enum": [ "null", "0", "1", "2"],
+                                "enum": [ null, 0, 1, 2],
```
Comment (karenkrieger) [inline VTEX - Payments Gateway API.json:5316]:
```suggestion
                                "enum": [null, 0, 1, 2],
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Payments Gateway API.json [hunk 6]
```diff
@@ -3081,153 +3083,166 @@
...
+                                            "type": "object",
+                                            "description": "Shipping information pertinent to the order.",
+                                            "properties": {
+                                                "address": {
+                                                    "type": "object",
+                                                    "description": "Address information.",
+                                                    "properties": {
+                                                        "addressType": {
+                                                            "type": "string",
+                                                            "description": "Type of address. For example, `Residential` or `Pickup`.",
```
[the remark this answers is not in the document]
Comment (ricardoaerobr) [reply VTEX - Payments Gateway API.json:3173]: Corrigido example para `Residential`.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**